.github/workflows/base-package-tests.yaml

```diff
@@ -13,7 +13,7 @@ on:
 jobs:
   package-tests:
     name: Package Tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Should we split this into a separate PR?

Reply:
removed